workers/unity/Packages/com.improbable.gdk.core/GameObjectRepresentation/GameObjectReferenceHandle.cs

```diff
@@ -1,6 +1,6 @@
 using Unity.Entities;
 
-namespace Playground
+namespace Improbable.Gdk.Core
```

Comment:
Please put these in a more specific namespace - applies to most files in this PR